packages/ihe-gateway/server/CodeTemplates/Common Library/File type/File type.js

```diff
@@ -86,7 +86,7 @@ function isLikelyTextFile(fileBuffer) {
 }
 
 function isLikelyXML(contents) {
-  if (contents && typeof contents === "string") {
+  if (contents) {
```

Comment:
Why not checking if its string?

If we really need to remove the check based on `typeof`, we should at least make sure whatever object we get it contains the functions we're calling, so we don't get a runtime error.

Reply:
So the issue was that the contents is an object based on the upstream changes. I can add typeof == "object"

Reply:
I added

Reply:
Wait, how is this an object, [the code](https://github.com/metriport/metriport/pull/1752/files#diff-0046a03c953ff3fa38d7ead8bdfcc4899c4009e699ef974df81fce8d0c3dd827R27-R28) is sending a string, isn't it?

And checking for `object` doesn't mean it will have `startsWith`, right?